tests/scripts/thread-cert/test_netdata_publisher.py

```diff
@@ -263,24 +263,45 @@ def test(self):
             self.verify_anycast_services(services)
 
         #---------------------------------------------------------------------------------
-        # DNS/SRP unicast entries
+        # DNS/SRP service data unicast entries
 
-        # Publish DNS/SRP unicast address on all routers, first using
-        # MLE-EID address, then change to use specific address. Verify
-        # that number of entries in network data is correct in each step
-        # and that entries are switched correctly.
         num = 0
         for node in routers:
-            node.netdata_publish_dnssrp_unicast_mleid(DNSSRP_PORT)
+            node.netdata_publish_dnssrp_unicast(DNSSRP_ADDRESS, DNSSRP_PORT)
             self.simulator.go(WAIT_TIME)
             num += 1
             services = leader.get_services()
             self.assertEqual(len(services), min(num, DESIRED_NUM_DNSSRP_UNCIAST))
             self.verify_unicast_services(services)
 
         for node in routers:
-            node.netdata_publish_dnssrp_unicast(DNSSRP_ADDRESS, DNSSRP_PORT)
+            node.srp_server_set_enabled(True)
+            self.simulator.go(WAIT_TIME)
+
+        self.assertEqual(sum(node.srp_server_get_state() == 'running' for node in routers),
+                         min(len(routers), DESIRED_NUM_DNSSRP_UNCIAST))
```

Comment:
Maybe fix the typo while we update these code parts?

```suggestion
                         min(len(routers), DESIRED_NUM_DNSSRP_UNICAST))
```

Reply:
Thanks. Fixed in the new push.